.github/workflows/markdown-link-check.yml

```
@@ -0,0 +1,15 @@
+name: Check Markdown links
+
+on: 
+  push:
+    branches: 
+      - master
+  schedule:
+    - cron: "0 18 * * *"
```

```suggestion
    - cron: "28 2 3,17 * *"
```
28 and 2 because I think a lot of other cron jobs are running at 0 6 ;)

Every 2 weeks because I think that should be adequate ;)